scripts/AccessControl.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-console */
```

Comment:
please remove this file from this pull request. it is a different task

Reply:
done